exporters/otlp/src/recordable.cc

```diff
@@ -18,19 +18,101 @@ void Recordable::SetIds(trace::TraceId trace_id,
 void Recordable::SetAttribute(nostd::string_view key,
                               const opentelemetry::common::AttributeValue &value) noexcept
 {
-  (void)key;
-  (void)value;
+  auto attribute = span_.add_attributes();
```

Comment:
nit: auto -> auto*. It is better to readability to use auto* instead of auto with pointers, although either would work.